packages/web/src/app/app.component.ts

```diff
@@ -64,7 +64,11 @@ export class AppComponent {
   currentRoute: PageLayout = 'standalonePage';
   isLandingPage = false;
 
-  constructor(private routerService: RouterService) {
+  constructor(
+    private routerService: RouterService,
+    viewportScroller: ViewportScroller,
+  ) {
+    viewportScroller.setOffset([0, 80]);
```

Comment:
This replaces the old `RouterModule.forRoot(..., {scrollOffset: [0, 80])`, as that method of defining a global scroll offset has been deprecated.

Reply:
https://stackoverflow.com/a/77498432